src/main/java/ru/svgogin/service/spark/dto/CompanyDto.java

```diff
@@ -0,0 +1,52 @@
+package ru.svgogin.service.spark.dto;
+
+import java.util.Date;
+
+public class CompanyDto {
+  private final String inn;
+  private final String ogrn;
+  private final String kpp;
+  private final String full_name_rus;
```

Comment:
Use camelCase in java.
```suggestion
  private final String fullNameRus;
```
You can find some guides here:
- https://google.github.io/styleguide/javaguide.html
- http://cr.openjdk.java.net/~alundblad/styleguide/index-v6.html
- https://www.oracle.com/java/technologies/javase/codeconventions-contents.html